app/jobs/appointment_reminder_job.rb

```diff
@@ -5,7 +5,7 @@ def perform(appointment)
     ReminderActivity.create!(
       appointment: appointment,
       owner: appointment.guider,
-      message: 'sent a reminder'
+      message: 'sent an appointment reminder email to the customer'
```

Comment:
I'm going to go through this code and remove the dependency on the `message` column, where appropriate.